transportable-udfs-trino/src/main/java/com/linkedin/transport/trino/utils/TrinoKeywordsConverter.java

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.trino.utils;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+/**
+ * Trino Keywords Converter maintains the Trino Keywords and related utility functions
+ */
+public final class TrinoKeywordsConverter {
+
+  private TrinoKeywordsConverter() {
+
+  }
+
+  /**
+   * Map of Trino Reserved Keywords with the mark to distinguish if the word is reserved SQL:2016 keyword ONLY
+   */
+  private static final Map<String, Boolean> RESERVED_KEYWORDS =
+      ImmutableMap.<String, Boolean>builder().put("ALTER", Boolean.FALSE)
```

Comment:
Do we need a Map? what does the `FALSE` mean?

Reply:
It was copied from Coral, I think Set should be enough, going to modify it.